apps/block_scout_web/assets/css/theme/_variables-non-critical.scss

```diff
@@ -1,5 +1,6 @@
 @import "theme/base_variables";
 @import "neutral_variables-non-critical";
```

Comment:
This should be probably removed. Please check if your fix is then still necessary.

Reply:
I'm not sure what you mean. So it can be removed. It is not necessary as long as we have our own definitions, else the pre-compile fails. But the "fix" is necessary in any case.
If you have some specific change in mind that I should try, please elaborate it. Else I've tested already many cases. Having a look at the other networks, that's what they do as well. Except that I reuse the definitions.

Reply:
So I took again the other networks as example. As in the call agreed on, I commented out the neutral colors. Moreover do I not attempt to import the non-critical colors, but simply clone them.